storage_manager_new.py

```diff
@@ -15,37 +15,20 @@ def clear_self_items_list_and_load_items_from_file(self):
             line = line.rstrip('\n')
             self.items.append(Item.create_instance_from_string(line))
 
-    def add_item(self):
-        # Collect new item data
-        new_items_data = []
-        while True:
-            new_item = input(
-                'Please provide new item data in order: category, type, model, quantity (Example: Sound, Mixer, Midas M32, 1).\n')
-            if new_item.lower() != 'menu':
-                new_items_data.append(new_item)
-            else:
-                break
-            ask_for_more = input(
-                'Would you like to add another item? Please type Y/N\n')
-            if ask_for_more.lower() == 'n':
-                break
-        # Create item instance and write to database file
-        for string_with_new_item_data in new_items_data:
-            try:
-                list_from_string_with_new_item_data = string_with_new_item_data.split(
-                    ', ')
-                quantity = int(list_from_string_with_new_item_data.pop(3))
-                new_item_data_without_quantity = ', '.join(
-                    list_from_string_with_new_item_data)
-                for i in range(1, (quantity + 1)):
-                    Item.create_instance_from_string_and_write_to_database_file(
-                        new_item_data_without_quantity)
-                    print(
-                        f'{new_item_data_without_quantity} was added successfully.')
-            except (IndexError, ValueError):
-                print('Error: adding item  was cancelled, please try again.')
-        print('')
-        self.clear_self_items_list_and_load_items_from_file()
+    def add_item(self, string_with_new_item_data):
+        try:
+            list_from_string_with_new_item_data = string_with_new_item_data.split(
+                ', ')
+            quantity = int(list_from_string_with_new_item_data.pop(3))
+            new_item_data_without_quantity = ', '.join(
+                list_from_string_with_new_item_data)
+            for i in range(1, (quantity + 1)):
+                Item.create_instance_from_string_and_write_to_database_file(
+                    new_item_data_without_quantity)
+            self.clear_self_items_list_and_load_items_from_file()
+            return True
+        except (IndexError, ValueError):
+            return False
```

Comment:
If there are multiple possibilities for this to fail, maybe we should give more context info when returning? Not simple bool but some message maybe?